docs/internals/contributing/writing-code/working-with-git.txt

```diff
@@ -281,12 +281,12 @@ Your pull request should now contain the new commit too.
 Note that the merger is likely to squash the review commit into the previous
 commit when committing the code.
 
-Working on a patch
-==================
+Working on a pull request
```

Comment:
I feel like this title is misleading "Testing a pull request locally" maybe (or branch/contribution)

Reply:
Yes, I agree. Went for "Testing a pull request locally", that seems the most accurate. 👍🏻